.github/workflows/nodejs.yml

```
@@ -0,0 +1,32 @@
+name: Node CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - stable*
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install, build
```

Let's e consistent on the steps case :see_no_evil: 
```suggestion
    - name: Install dependencies & build
```